frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions.styled.jsx

```diff
@@ -13,13 +14,20 @@ export const ExpressionList = styled.ul`
 `;
 
 export const ExpressionListItem = styled.li`
-  padding-top: 5px;
-  padding-bottom: 5px;
+  display: flex;
+  align-items: center;
+  padding: 5px 1rem;
```

Comment:
nit: use `0.3125rem` for consistency. e.g. https://github.com/metabase/metabase/blob/5199fb4de20b2cd72ff25ad637290d822863c570/frontend/src/metabase/visualizations/components/ChartClickActions.styled.tsx#L55

Reply:
Fixed, thanks!